.gitlab-ci.yml

```diff
@@ -64,4 +64,10 @@ pytest:
   stage: test
   extends: .base
   script:
-    - python -m pytest
+    - python -m pytest -m "not functional"
+
+functional_tests:
+  stage: test
+  extends: .base
+  script:
+    - python -m pytest -m functional --durations=0 -v
```

Comment:
Why are they separated in their own module, can't everything be together ?

Reply:
I am accustomed to have a clear separation between unit, functional and integration tests during deployments but it doesn't have to. I could add a dependency to it, so that functional tests do not run unless unit tests are passing.